docs/self-managed/operational-guides/monitoring/metrics.md

```diff
@@ -0,0 +1,222 @@
+mvn dependency:copy -Dartifact=io.micrometer:micrometer-registry-datadog:1.14.4 -Dtransitive=false -DoutputDirectory=./lib
+```
+
+:::note
+Make sure the version is the same as the Micrometer version used by Camunda. You can find this out by checking the distribution artifact on
+[Maven Central](https://central.sonatype.com/artifact/io.camunda/camunda-zeebe/dependencies).
+
+Make sure to select the distribution version you're using, then filter for `micrometer` to get the expected Micrometer version.
+:::
+
+### Customizing metrics
+
+You can modify and filter the metrics exposed in Camunda via configuration.
+
+#### Common tags
+
+[Tags provide a convenient way of aggregating metrics over common attributes](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.customizing.common-tags).
+Via configuration, you can ensure that all metrics for a specific instance of Camunda share common tags. For example, let's say you deploy two different clusters, and want to differentiate them.
+
+The first one could be configured as:
+
+```yaml
+management:
+  metrics:
+    tags:
+      cluster: "foo"
+```
+
+And the second one as:
+
+```yaml
+management:
+  metrics:
+    tags:
+      cluster: "bar"
+```
+
+#### Filtering
+
+[You can additionally disable certain metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.customizing.per-meter-properties).
+This can be useful for high cardinality metrics which you do not care for, but which may end up being expensive to store in your target system.
+
+To filter a metric called `zeebe.foo`, you would configure the following property:
+
+```yaml
+management:
+  metrics:
+    enable:
+      zeebe:
+        foo: false
+```
+
+:::note
+Filtering applies not only to direct name matches (e.g. `zeebe.foo`), but as a prefix. Meaning any metric starting with the prefix `zeebe.foo` in the example
+above would also be filtered out, and wouldn't be exported.
+:::
+
+## Available metrics
+
+[Spring already exposes various metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.supported), some of which will be made available
+through Camunda:
+
+- [JVM metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.supported.jvm)
+- [System metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.supported.system)
+- [Application startup metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.supported.application-startup)
+- [Logger metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.supported.logger)
+- [Spring MVC metrics](https://docs.spring.io/spring-boot/reference/actuator/metrics.html#actuator.metrics.supported.spring-mvc)
+
+Additionally, Camunda will expose several custom metrics, all of which are prefixed with either `zeebe`, `atomix`, `operate`, `tasklist`, or `optimize`.
```

Comment:
:wrench: I'd try to be less specific here:
1. This list is already incomplete, for example the Connector runtime exposes metrics under `camunda.connector`
2. It's difficult to keep this list up to date

Reply:
I agree, but I think it's useful to give them a hint of how they can figure out what metrics are available, at least until we have https://github.com/camunda/camunda-docs/issues/5194

I'm not sure what we can do to be less specific, other than specify we expose several custom metrics without saying which ones? :smile:

Reply:
Maybe just say "Additionally, Camunda will expose several custom metrics, most of them under the  `zeebe`, `atomix`, `operate`, `tasklist`, and `optimize`. prefixes."?